start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloReloadingVFSClassLoader.java

```diff
@@ -183,12 +184,23 @@ public AccumuloReloadingVFSClassLoader(String uris, FileSystemManager vfs,
     monitor.setDelay(monitorDelay);
     monitor.setRecursive(false);
     for (FileObject file : pathsToMonitor) {
-      monitor.addFile(file);
+      addFileToMonitor(file);
       log.debug("monitoring {}", file);
     }
     monitor.start();
   }
 
+  private void addFileToMonitor(FileObject file) {
+    try {
+      monitor.addFile(file);
+    } catch (RuntimeException re) {
+      if (re.getMessage().contains("files-cache"))
+        log.error("There is no implementation for files-cache in VFS2");
+      else
+        log.error(re.getMessage());
```

Comment:
Not sure, but maybe we should re-throw here?

```suggestion
        throw re;
```

At the very least, we should include the stack trace.

```suggestion
        log.error("Unknown error adding {} to VFS monitor", file, re);
```